lib/cn/wbt_builder.c

```diff
@@ -89,7 +88,7 @@ struct wbb {
 struct key_stage_entry_leaf {
     u32 kmd_off;
     u16 klen;
-    u8  kdata[] HSE_ALIGNED(sizeof(void *));
+    u8  kdata[] HSE_ALIGNED(CHAR_BIT);
```

Comment:
This changes the alignment.  It was word aligned, is now byte aligned.  Maybe just add a u16 pad before kdata?  Or don't align or pad.  Who can really measure the performance benefit of this change?

Reply:
Doesn't sizeof(CHAR_BIT) == sizeof(void *)?

Reply:
Oh... I guess it does.  But why align an address to the number of bits in a char?  That doesn't make any sense.  This alignment directive is used to align things so they can be loaded from memory more efficiently right?  Seems like "pointer aligned" is what we want.

Reply:
CHAR_BIT is 8, but it makes no sense to use in this context.  We want kdata[] to be ptr aligned so that the linker will call the 8-byte aligned memcpy/memcmp functions, which outperform the byte-aligned variants.   For that we want the data to actually have 8-byte alignment rather than 6-byte alignment, hence the coerced alignment.

Reply:
Thanks for the explanation